reconcilliation-test/README.md

```diff
@@ -0,0 +1,20 @@
+
+What
+===
+
+This is a small script that checks whether the data in the cloud foundry database is similar to the data in the billing database about what services exist.
```

Comment:
So, in english, what this does is check that all currently existing service instances older than 4h have billing events in the last 2d, and that there aren't *too many* service instances that have billing events in the last 2 days that don't seem to currently exist? If correct, this might be nice to include here.